src/ophyd_async/core/_detector.py

```diff
@@ -267,6 +274,13 @@ async def read(self) -> dict[str, Reading]:
     async def describe(self) -> dict[str, DataKey]:
         return self._describe
 
+    # @staticmethod
+    # def ensure_trigger_info_exists(trigger_info: TriggerInfo | None) -> TriggerInfo:
+    #     # make absolute sure we realy have a valid TriggerInfo ... mostly for pylance
+    #     if trigger_info is None:
+    #         raise RuntimeError("Trigger info must be set before calling this method.")
+    #     return trigger_info
+
```

Comment:
Please can you remove the commented out code

Reply:
sorry, totally overlooked that
done a3fa08d8d7b84681d18139e321a8ba2e1e648019